content/en/tracing/guide/slowest_request_daily.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Investigate the slowest request of a day and combine infrastructure data to find the root cause
+kind: guide
+disable_toc: true
+further_reading:
+- link: "tracing/setup/"
+  tag: "Documentation"
+  text: "Learn how to setup APM tracing with your application"
+- link: "tracing/visualization/services_list/"
+  tag: "Documentation"
+  text: "Discover the list of services reporting to Datadog"
+- link: "tracing/visualization/resource"
+  tag: "Documentation"
+  text: "Dive into your resource performance and traces"
+- link: "tracing/visualization/trace"
+  tag: "Documentation"
+  text: "Understand how to read a Datadog Trace"
+---
+
+{{< img src="tracing/guide/slowest_request_daily/slowest_trace_1.gif" alt="Identifying the slowest trace and finding the Host metrics for it" responsive="true" style="width:90%;">}}
+
+Using Datadog APM you can easily investigate the performance of your various endpoints, identify slow requests and investigate the root cause of their latency by having infrastructure metrics alongside the distributed trace. This example shows the slowest trace of the day for an e-commerce checkout endpoint and how it slows down because of high CPU usage.
```

Comment:
```suggestion
With Datadog APM, you can easily investigate the performance of your endpoints, identify slow requests, and investigate the root cause of latency issues. This example shows the slowest trace of the day for an e-commerce checkout endpoint and how it slows down because of high CPU usage.
```